thyrel-web/src/components/home/PlayerForm.tsx

```diff
@@ -41,13 +44,15 @@ export default function PlayerForm({ identifier }: { identifier?: string }) {
   }
 
   function onJoin(identifier: string) {
+    setLoading(true);
     client<Player>(`room/join/${identifier}`, {
       data: {
         username: username || defaultUsername,
         avatarUrl: String(ppIndex),
       },
       method: 'PATCH',
     }).then((player: Player) => {
+      setLoading(false);
```

Comment:
rajouté un `.catch` qui set aussi loading to false (ou un deuxième param au `.then`) car ici si la requete crash ça load à l'infini. Peut-être même en profité pour ajouté une notification